test/e2e/scenarios/Cluster/cluster_update_with_tags.yaml

```diff
@@ -53,12 +53,12 @@ steps:
             value: "test_value_1"
           - key: "test_key_2"
             value: "test_value_2"
```

Comment:
nit: Asserting `k8s` tags are on the spec isn't necessary. We know they are on the spec because lines 38-42 patched them onto it.

Reply:
How about other fields? If we do patch some fields, is it also not necessary to assert corresponding k8s fields?

Reply:
The only time a spec field would change is if there is what we call "late initialisation" - the memorydb service changes the value, or provides a default value. In those cases it would be good to test that the service updated it as we would expect, and the spec now contains those values. Otherwise I think it's not necessary to test

Reply:
That seems reasonable to me